recipe/meta.yaml

```diff
@@ -49,8 +49,7 @@ test:
     - pip
   commands:
     - pip check
-    - python -m scalene --cli test/original/bm_mdp.py  # [not win]
-    - python -m scalene --cli test\\original\\bm_mdp.py  # [win]
+    - scalene --cli test/original/bm_mdp.py
```

Comment:
This won't work for Windows because the slashes are in the "wrong" direction. You need both the [win] and [not win] variants in the original.

Reply:
Windows is actually agnostic to whether you use forward or backslashes. I've always found it causes fewer issues to just always use forward slashes.

Reply:
That is not my experience (in Command Prompt):

<img width="478" alt="Screenshot 2023-10-11 at 10 14 25 PM" src="https://github.com/conda-forge/scalene-feedstock/assets/1612723/26f7350f-9e1e-4e69-9f70-c2011f00b65d">

Reply:
You also don't even need to use a drive letter:
```
❯ ls C:\temp

    Directory: C:\temp

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a---          12/10/2023  1:15 PM              0 a.txt


❯ ls C:/temp

    Directory: C:\temp

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a---          12/10/2023  1:15 PM              0 a.txt


❯ ls /temp

    Directory: C:\temp

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a---          12/10/2023  1:15 PM              0 a.txt
```

Reply:
But if it works in this context, great!